documentation/releases.md

```diff
@@ -0,0 +1,13 @@
+# Releases
+
+We use a GitHub Action to publish a crate named `prio` to [crates.io](https://crates.io). To cut a
+release and publish:
+
+- Bump the version number in `Cargo.toml` to e.g. `1.2.3` and merge that change to `main`
+- Tag that commit on main as `v1.2.3`, either in `git` or in [GitHub's releases UI](releases).
```

Comment:
I wasn't previously aware of [reference-style links](https://www.markdownguide.org/basic-syntax/#reference-style-links) -- I think you might need to change the syntax for these to use square brackets for both portions of the link.

That is:
```suggestion
- Tag that commit on main as `v1.2.3`, either in `git` or in [GitHub's releases UI][releases].
```